src/splinterdb.c

```diff
@@ -686,3 +692,10 @@ splinterdb_stats_reset(splinterdb *kvs)
 {
    trunk_reset_stats(kvs->spl);
 }
+
+// TRUE if Splinter's task system was configured to use background threads.
+bool
+splinterdb_task_system_uses_bg_threads(splinterdb *kvs)
```

Comment:
Deleted this new lookup function.

Reply:
I still see it in the diff here.

Reply:
Yes, my omission. Thanks, for catching this ... Deleted.